helm/vitess/templates/_vtworker.tpl

```diff
@@ -0,0 +1,81 @@
+{{- define "vtworker" -}}
+{{- $ := index . 0 -}}
+{{- $cell := index . 1 -}}
+{{- with index . 2 -}}
+{{- $0 := $.Values.vtworker -}}
+# vtworker
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: vtworker
+spec:
+  replicas: {{.replicas | default $0.replicas}}
+  template:
+    metadata:
+      labels:
+        component: vtworker
+        app: vitess
+      annotations:
+        pod.beta.kubernetes.io/init-containers: '[
+{{ include "init-vtdataroot" (.image | default $0.image) | indent 10 }},
+          {
+            "name": "init-vtworker",
+            "image": {{.image | default $0.image | quote}},
+            "imagePullPolicy": "IfNotPresent",
+            "volumeMounts": [
+              {
+                "name": "config",
+                "mountPath": "/mnt/config"
+              },
+            ]
+          }
+        ]'
+    spec:
+      containers:
+        - name: vtworker
+          image: {{.image | default $0.image | quote}}
+          livenessProbe:
+            httpGet:
+              path: /debug/vars
+              port: 17032
+            initialDelaySeconds: 30
+            timeoutSeconds: 5
+          volumeMounts:
+            - name: syslog
+              mountPath: /dev/log
+            - name: vtdataroot
+              mountPath: /vt/vtdataroot
+            - name: web
```

Comment:
Similarly, there are no web/certs volume mounts so I think you can delete these too.